intercepting-filter/src/main/Filter.java

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I would like to see more documentation. The filters have an important role in the pattern, right?